### DISCUSSION TITLE
Question about formulations tab

### DISCUSSION THREAD

**Discussion by kyun-gran** [Open-Systems-Pharmacology/Forum#1587] [Questions & Problems]
Hi, i'm a  graduate student studying PK-SIM.
I have some questions about formulations tab.
We want to create a Table under formulations tab using data with solute pH 1.2.
Here i have two quesition about it.
1. Is there a default pH set into this tab?
2. How can i modify the solute pH under formulations tab or other options?

I'm looking forward to hearing from you.
Thank you

**Comment by Yuri05** [comment]
Is this what you are looking for?
https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-compounds-definition-and-work-flow#intestinal-solubility-as-table-function-of-ph

![grafik](https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/7d0a51a6-6096-407c-9df6-5d679849aa2a)

**Reply by SCZ1174** [reply]
WOW, a new discovery, thanks!

**Reply by kyun-gran** [reply]
![캡처](https://github.com/Open-Systems-Pharmacology/Forum/assets/75706166/bfe5303f-5204-450f-8f1e-e689bb1b5ac7)
First of all, thank you for your reply.
That was a question on the formulation tab.
Can't the solute pH be modified in the formulation tab as shown in the picture here?

I melted api at PH 1.2, and as a result, I would like to enter the DISSOLUTON result over time.

**Reply by SCZ1174** [reply]
It's simply a matter of adding an extra set when you set the solubility of the compound parameter, and then following the actions below should get you where you need to be.
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/86552551/66b72613-57e6-44e3-b38d-a134b220c49d)
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/86552551/836ae66f-0484-4db3-9b25-029fc8b18c55)
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/86552551/50ecc0db-4011-412a-a1c8-df36b6ee3727)
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/86552551/9d43580c-facc-4bb8-9260-f361d830c085)

**Reply by kyun-gran** [reply]
Yes, I understand.
However, I would like to add the result of the dissolution data over time.
However, there is no place to enter time data in the displayed figure. 
It appears that there are only places where you enter solubility and pH data.

**Reply by RobinM92** [reply]
Hi @kyun-gran,

You can do that in the formulation tab by adding a dissolution table: 

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/30694339/73199d6c-b149-4c3a-a178-35fbd455fb5d)

**Reply by kyun-gran** [reply]
Yes, I did it first.
However, there is no space to write down information about pH.
That's why I asked the question.
My question is as follows.
We want to create a Table under formulations tab using data with solute pH 1.2.
Here i have two quesition about it.

Is there a default pH set into this tab?
How can i modify the solute pH under formulations tab or other options?
